src/types.rs

```diff
@@ -66,11 +67,16 @@ pub struct OutputId(Arc<str>);
 pub struct InputId(Arc<str>);
 
 #[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
-pub struct OutputScene {
+#[serde(deny_unknown_fields)]
+pub struct UpdateOutputRequest {
     pub output_id: OutputId,
-    pub scene: Component,
+    pub video: Option<Component>,
+    pub audio: Option<AudioComposition>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema)]
+pub struct AudioComposition(Vec<InputId>);
```

Comment:
Do not make it just vector of strings, at least make it sth like `[{"input_id": "id"}, ...]`. With this API when we need to add any option per input we will have to make a breaking change.